app/javascript/packs/verification_tool.js

```diff
@@ -15,6 +16,7 @@ function removeInstallPrompt() {
   }
 }
 
+Vue.component('action_wrapper', actionWrapper)
```

Comment:
I think this should either be in PascalCase or kebab-case, according to https://vuejs.org/v2/style-guide/#Component-name-casing-in-templates-strongly-recommended

Reply:
Changed in 7f31b2e